Tasks/GruntV0/task.loc.json

```diff
@@ -16,8 +16,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 166,
-    "Patch": 2
+    "Minor": 182,
```

Comment:
Are these redundant changes related to node 10 migration?

Reply:
This branch is based on node10 migration branch. I was going to merge it first